### DISCUSSION TITLE
Add support for Woocommerce B2B Plugin

### DISCUSSION THREAD

**Discussion by freefugga** [inspireui/support#5123] [Working on Features]
Hi,

Please add support for Woocommerce B2B Plugin. It enables us to show different prices of items to different user roles.

This is the module. I am requesting this one as this also has a Sales Agent Module which would be helpful for wholesale businesses which hire Sales Executives and the dealers are assigned under such executives.

https://codecanyon.net/item/woocommerce-b2b/21565847?s_rank=2

**Comment by HanielQC** [comment]
FluxStore 3.8.1 https://1.envato.market/mKdNe is supporting WooCommerce Wholesale https://youtu.be/fgWtlo4lDZ4
Thank you
